### PR TITLE
No dask-distributed dependency

### PR DESCRIPTION
**Pull request by hmaarrfk**
We don't have a `dask-array` dependency on conda-forge, so you typically have to add the `cytoolz` and `numpy` dependency manually.

Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by hmaarrfk** [comment]
`distributed` is quite heavy and slows down import time for some something.

File: recipe/meta.yaml
Comment (jaimergp) [inline recipe/meta.yaml:127]: Any reason not to add these explicitly in each output? Is it just a simplification or are we trying to prevent some faulty behavior?

Comment (hmaarrfk) [reply recipe/meta.yaml:127]: just a simplification. I think the bot also has trouble with either "build" or "number" and I never remember which one it fails to bump automatically.

**Comment by jaimergp** [comment]
I think this is ok. I just wish `dask-core` had `run_constrained` entries for numpy and cytoolz so we don't have to check whether those are up-to-date with our own requirements (maybe this is ok to add in `dask-core-feedstock`?). I'll defer to the napari maintainers here.

**Comment by hmaarrfk** [comment]
> I think this is ok. I just wish `dask-core` had `run_constrained` entries for numpy and cytoolz so we don't have to check whether those are up-to-date with our own requirements (maybe this is ok to add in `dask-core-feedstock`?). 

This is one of the first issues I opened at conda-forge, there just wasn't enough enthusiasm by the dask team to take it on at the time
https://github.com/conda-forge/dask-feedstock/issues/46

**Comment by hmaarrfk** [comment]
@conda-forge/napari any objections to this as is.

distributed is really a heavy dependency for users to "load" accidentally on conda-forge.

**Review by jni — APPROVED** [review]
fine by me

**Review by GenevieveBuckley — APPROVED** [review]
This is fine by me, too.

**Pull request merged**
